src/App.tsx

```diff
@@ -50,15 +35,17 @@ export const App: React.FC = () => {
           <div className="tile is-parent">
             <div className="tile is-child box is-success">
               <div className="block">
-                <UserSelector value={author} onChange={setAuthor} />
+                <UserSelector />
               </div>
 
               <div className="block" data-cy="MainContent">
-                {!author && <p data-cy="NoSelectedUser">No user selected</p>}
+                {!selectedUser && (
+                  <p data-cy="NoSelectedUser">No user selected</p>
+                )}
 
-                {author && !loaded && <Loader />}
+                {selectedUser && loaded && <Loader />}
```

Comment:
The logic for rendering the `Loader` component is incorrect. The `Loader` should be displayed when the data is loading, not when it is already loaded. Consider changing the condition to `selectedUser && !loaded` to show the loader while data is being fetched.